doc/microsite/src/main/tut/distage/distage-framework-docker.md

```diff
@@ -19,3 +20,487 @@ libraryDependencies += "io.7mind.izumi" %% "distage-framework-docker" % "$izumi.
 ```
 
 @@@
+
+
+### Overview
+
+Use of `distage-framework-docker` generally follows:
+
+- Create @scaladoc[`ContainerDef`](izumi.distage.docker.ContainerDef)s for the containers the
+  application requires
+- Create a module that declares the container component
+- Include the @scaladoc[`DockerSupportModule`](izumi.distage.docker.modules.DockerSupportModule) in
+  the application's modules
+
+First, the required imports:
+
+```scala mdoc:silent
+import izumi.distage.docker.ContainerDef
+import izumi.distage.docker.Docker
+import izumi.distage.model.definition.ModuleDef
+import izumi.reflect.TagK
+```
+
+#### Create a `ContainerDef`
+
+The @scaladoc[`ContainerDef`](izumi.distage.docker.ContainerDef) is a trait used to define a docker
+container. Extend this trait and provide and implementation of the `config` method.
+
+The required parameters of `Config` are:
+
+- `image` - the docker image to use
+- `ports` - ports to map on the docker container
+
+See @scaladoc[`Docker.ContainerConfig[T]`](izumi.distage.docker.Docker.ContainerConfig) for
+additional parameters.
+
+Example [postgres](https://hub.docker.com/_/postgres/) container definition:
+
+```scala mdoc:silent
+object PostgresDocker extends ContainerDef {
+  val primaryPort: Docker.DockerPort = Docker.DockerPort.TCP(5432)
+
+  override def config: Config = {
+    Config(
+      image = "library/postgres:12.2",
+      ports = Seq(primaryPort),
+      env = Map("POSTGRES_PASSWORD" -> "postgres"),
+    )
+  }
+}
+```
+
+#### Declare Container Components
+
+To use this container a module that declares this component is required:
+
+##### `make`
+
+```scala mdoc:silent
+class PostgresDockerModule[F[_]: TagK] extends ModuleDef {
+  make[PostgresDocker.Container].fromResource {
+    PostgresDocker.make[F]
+  }
+}
+```
+
+##### `modifyConfig`
+
+The @scaladoc[`DockerProviderExtensions`](izumi.distage.docker.DockerContainer.DockerProviderExtensions)
```

Comment:
This link does not resolve unfortunately, probably has to be `izumi.distage.docker.DockerContainer$$DockerProviderExtensions`